third_party/cpgfunctionEP-0.3.2/include/cpgfunction/statistics.h

```diff
@@ -0,0 +1,20 @@
+//
```

Comment:
Statistics module that is used in running tests. Contains RMSE percentage function.